spec/js-api/value/calculation.d.ts.md

```diff
@@ -75,18 +75,13 @@ function boundary rather than when a calculation is constructed.
 ```ts
 import {List, ValueObject} from 'immutable';
 
-import {Value, SassNumber, SassString} from '../spec/js-api/value';
+import {Value, SassNumber, SassString} from './index';
 ```
 
 ## Types
 
 ### `Value`
```

Comment:
Similarly to the above: the proposal includes changes to _every_ type that gets updated, but the calculation specification should only define the `SassCalculation` itself and the types it uses.